sections/difficulty.tex

```diff
@@ -0,0 +1,29 @@
+% !TeX spellcheck = en_US
+\addsection{Difficulty}{\skills/water_magic.png}
```

Comment:
I don't like water magic here. I suggest one of the following: stone skin, haste, fire magic. In my head, I have a slight preference for stone skin, but please check them and choose what looks best.

Reply:
It's fine.